src/shared/killall.c

```
@@ -22,10 +23,52 @@
 #include "string-util.h"
 #include "terminal-util.h"
 
-static bool ignore_proc(pid_t pid, bool warn_rootfs) {
+static bool argv_has_at(pid_t pid) {
         _cleanup_fclose_ FILE *f = NULL;
         const char *p;
         char c = 0;
+
+        p = procfs_file_alloca(pid, "cmdline");
+        f = fopen(p, "re");
+        if (!f)
+                return true; /* not really, but has the desired effect */
```

debug log msg please when you decide to ignore errors

---

(i understand this is just moved around older code. but if you touch it, please update to our newest coding style)